playbooks/templates/rax-maas/ironic_capacity_check.yaml.j2

```diff
@@ -0,0 +1,20 @@
+{% set label = "ironic_api_local_check" %}
```

Comment:
Let's get someone more familiar with MaaS checks to thoroughly review this.

Also, do we need to support private network checks? cc @briancurtin